packages/core/admin/admin/src/pages/ProfilePage/components/Password/index.js

```diff
@@ -0,0 +1,217 @@
+import React, { useState } from "react";
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  Typography,
+  Box,
+  Grid,
+  GridItem,
+  Flex,
+  FieldAction,
+  TextInput,
+} from '@strapi/design-system';
+import { Eye, EyeStriked } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+
+const PasswordInput = styled(TextInput)`
+  ::-ms-reveal {
+    display: none;
+  }
+`;
+
+const FieldActionWrapper = styled(FieldAction)`
+  svg {
+    height: 1rem;
+    width: 1rem;
```

Comment:
Can you please use the `pxToRem` util from the hp.

Reply:
ok done